frontend/public/components/dashboard/project-dashboard/status-card.tsx

```diff
@@ -1,22 +1,57 @@
 import * as React from 'react';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector } from 'react-redux';
 import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
 import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
 import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
 import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
 import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
 import { Status } from '@console/shared';
+import { RootState } from '@console/internal/redux';
+import { getActiveNamespace } from '@console/internal/reducers/ui';
+import {
+  DashboardsOverviewHealthResourceSubsystem,
+  DashboardsOverviewHealthSubsystem,
+  isDashboardsOverviewHealthResourceSubsystem,
+  isDashboardsOverviewHealthSubsystem,
+  useExtensions,
+} from '@console/plugin-sdk';
 import { ProjectDashboardContext } from './project-dashboard-context';
+import { ResourceHealthItem } from '../dashboards-page/cluster-dashboard/health-item';
+
+import './status-card.scss';
 
 export const StatusCard: React.FC = () => {
   const { obj } = React.useContext(ProjectDashboardContext);
+  const filterSubsystems = (subsystems: DashboardsOverviewHealthSubsystem[]) =>
+    subsystems.filter(isDashboardsOverviewHealthResourceSubsystem);
+  const subsystemExtensions = useExtensions<DashboardsOverviewHealthSubsystem>(
+    isDashboardsOverviewHealthSubsystem,
+  );
+  const subsystem = React.useMemo(
+    () =>
+      filterSubsystems(subsystemExtensions).find(
+        (s) => s.properties.title === 'Image Vulnerabilities',
+      ),
+    [subsystemExtensions],
+  ) as DashboardsOverviewHealthResourceSubsystem;
+  const namespace = useSelector((state: RootState) => getActiveNamespace(state));
```

Comment:
Isn't we can use `obj.metadata.name` instead of `useSelector` for the namespace?

Reply:
yes, `namespace` is available in `obj.name`
updated to use `obj.name` for namespace
Thanks.